azure-iot-hub/azure/iot/hub/iothub_amqp_client.py

```diff
@@ -0,0 +1,72 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import logging
+import os
+import sys
+import base64
+import time
+import hashlib
+import hmac
+from uuid import uuid4
+import urllib
```

Comment:
tests are failing in 2.7 due to absence of a method in this module. 
This should be imported as `import six.moves.urllib as urllib` and i think tests would pass